Won't install

chintan@chintan-vm ~/Angular2Files> npm install
npm WARN package.json Angular2Files@1.0.0 No description
npm WARN package.json Angular2Files@1.0.0 No repository field.
npm WARN package.json Angular2Files@1.0.0 No README data
npm WARN package.json Angular2Files@1.0.0 No license field.
npm WARN peerDependencies The peer dependency es6-shim@^0.33.3 included from angular2 will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm ERR! Linux 3.19.0-15-generic
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install"
npm ERR! node v0.12.9
npm ERR! npm  v2.14.9
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package es6-shim@0.33.0 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer angular2@2.0.0-beta.0 wants es6-shim@^0.33.3

npm ERR! Please include the following file with any support request:
npm ERR!     /home/chintan/Angular2Files/npm-debug.log

--- a/package.json
+++ b/package.json
@@ -10,7 +10,7 @@
 		"angular2": "2.0.0-beta.0",
 		"systemjs": "0.19.6",
 		"es6-promise": "^3.0.2",
-		"es6-shim": "0.33.0",
+		"es6-shim": "^0.33.0",
 		"reflect-metadata": "0.1.2",
 		"rxjs": "5.0.0-beta.0",
 		"zone.js": "0.5.10"

